### DISCUSSION TITLE
What this?

### DISCUSSION THREAD

**Discussion by SPAWNRYS-ban** [ppy/osu#22815] [Q&A]
![osu!_fDgSri3DGu](https://user-images.githubusercontent.com/68144848/223819891-9fc4a286-32ee-486f-9c5f-b74f92883c03.png)
What is this purple square that keeps appearing to me? And it seems that after the map is open and has been passed
But I'm not sure.

**Comment by bdach** [comment]
Do you have any game overlays active? (discord, nvidia, whatever else)

**Reply by SPAWNRYS-ban** [reply]
no

**Reply by Joehuu** [reply]
A quick Google search of "purple square top left" says that it's Discord, can you try disabling the overlay temporarily or relaunching/closing Discord?
